apps/condo/domains/ticket/constants/common.js

```diff
@@ -3,7 +3,7 @@ const PDF_REPORT_WIDTH = 1600
 const DATE_DISPLAY_FORMAT = 'DD.MM.YYYY'
 const TICKET_REPORT_DAY_GROUP_STEPS = ['day', 'week']
 const TICKET_REPORT_TABLE_MAIN_GROUP = ['categoryClassifier', 'executor', 'assignee']
-const REQUIRED_TICKET_FIELDS = ['property', 'details', 'placeClassifier', 'categoryClassifier', 'deadline']
+const REQUIRED_TICKET_FIELDS = ['property', 'details', 'placeClassifier', 'categoryClassifier', 'deadline', 'assignee']
```

Comment:
`assignee` is not required anymore
```suggestion
const REQUIRED_TICKET_FIELDS = ['property', 'details', 'placeClassifier', 'categoryClassifier', 'deadline']
```